src/cfnlint/core.py

```diff
@@ -191,13 +188,15 @@ def run_checks(filename, template, rules, regions, mandatory_rules=None):
             raise InvalidRegionException(msg, 32)
 
     matches = []
-
+    """Pre-Processing"""
```

Comment:
This comment looks weird, why not `# Pre-processing`?

Reply:
Can add on for next PR